docs/README.md

```diff
@@ -102,22 +100,33 @@ To reproduce the use cases described in [Figure 2](https://docs.google.com/drawi
 
 ## Cite Us
 
-If you use the BioImage.IO Chatbot in your research, please cite us: <a href="https://zenodo.org/records/10032227" target="_blank"><img id="record-doi-badge" data-target="[data-modal='10.5281/zenodo.10032228']" src="https://zenodo.org/badge/DOI/10.5281/zenodo.10032228.svg" alt="10.5281/zenodo.10032228"></a>
-
+If you use the BioImage.IO Chatbot in your research, please cite us:
 
 ```
-Lei, W., Fuster-Barceló, C., Muñoz-Barrutia, A., & Ouyang, W. (2023). 🦒BioImage.IO Chatbot: A Personalized Assistant for BioImage Analysis Augmented by Community Knowledge Base (0.1.0). Zenodo. https://doi.org/10.5281/zenodo.10032228
-```
-
-You can also reference our preprint available on arXiv: [![arXiv](https://img.shields.io/badge/arXiv-2310.18351-red.svg)](https://arxiv.org/abs/2310.18351) 
-```
-Lei, W., Fuster-Barceló, C., Muñoz-Barrutia, A., & Ouyang, W. (2023). 🦒BioImage.IO Chatbot: A Personalized Assistant for BioImage Analysis Augmented by Community Knowledge Base. arXiv. https://arxiv.org/abs/2310.18351
+Lei, W., Fuster-Barceló, C., Reder, G. et al. BioImage.IO Chatbot: a community-driven AI assistant for integrative computational bioimaging. Nat Methods 21, 1368–1370 (2024). https://doi.org/10.1038/s41592-024-02370-y
 ```
 
 ## Acknowledgements
 
+The BioImage.IO Chatbot is greatly enriched by the valuable resources contributed by the global bioimage analysis community. We extend our deepest gratitude to all authors and contributors listed in the [knowledge base manifest](https://github.com/bioimage-io/bioimageio-chatbot/blob/main/knowledge-base-manifest.yaml), especially the original authors of the materials integrated into our extensions. Their work has significantly enhanced the capabilities of this chatbot, providing users with high-quality information and tools.
+
```

Comment:
Would it be possible to list the original authors of re-used materials here and the respective licenses? E.g. materials reused under a CC-BY license actually require you to acknowledge the authors by mentioning them and adding a link to the license. Read more: https://creativecommons.org/licenses/by/4.0/deed.en

Reply:
Thank you for your suggestion. Our plan is to store the authors, licenses, and references information in the YAML file, which will be linked in the README. This approach isn’t about downplaying the importance of the authors but rather ensuring the list remains maintainable, given that it could become extensive.

More importantly, we’ll also display this information directly in the chatbot interface, making it even more visible to a broader audience. This way, we ensure proper acknowledgment in a way that’s both practical and effective.